### PR TITLE
void

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -26,29 +26,35 @@ Make sure you get a `:thumbsup`, `:+1`, or `LGTM` from another collaborator befo
...
-We recommend creating a folder somewhere manually called `node_modules`. For example in `~/code/node_modules`. Clone all of your git copies of modules that you want to work on into here, so for example:
+Create a folder somewhere manually called `node_modules`. For example in `~/node_modules`. Clone all of your git copies of modules that you want to work on into here, so for example:
 
-- `~/code/node_modules/dat`
-- `~/code/node_modules/hyperdrive`
-
-When you run `npm install` inside of `~/code/node_modules/dat`, dat will get its own copy of `hyperdrive` (one if its dependencies) inside `~/code/node_modules/dat/node_modules`. However, if you encounter a bug in hyperdrive that you need to fix, but you want to test your fix in dat, you want dat to use your git copy of hyperdrive at `~/code/node_modules/hyperdrive` and not the npm copy of hyperdrive at `~/code/node_modules/dat/node_modules/hyperdrive`.
+```sh
+mkdir node_modules
+cd node modules
```
Comment (danfinlay) [inline CONTRIBUTING.md]: should be `node_modules`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -26,29 +26,35 @@ Make sure you get a `:thumbsup`, `:+1`, or `LGTM` from another collaborator befo
...
+git clone https://github.com/MetaMask/metamask-extension
+git clone https://github.com/MetaMask/provider-engine
+# clone more repos...
+cd metamask-extension
```
Comment (danfinlay) [inline CONTRIBUTING.md]: should be `cd ../metamask-extension`

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -26,29 +26,35 @@ Make sure you get a `:thumbsup`, `:+1`, or `LGTM` from another collaborator befo
...
-- `~/code/node_modules/dat`
-- `~/code/node_modules/hyperdrive`
-
-When you run `npm install` inside of `~/code/node_modules/dat`, dat will get its own copy of `hyperdrive` (one if its dependencies) inside `~/code/node_modules/dat/node_modules`. However, if you encounter a bug in hyperdrive that you need to fix, but you want to test your fix in dat, you want dat to use your git copy of hyperdrive at `~/code/node_modules/hyperdrive` and not the npm copy of hyperdrive at `~/code/node_modules/dat/node_modules/hyperdrive`.
+```sh
+mkdir node_modules
+cd node modules
+git clone https://github.com/MetaMask/metamask-extension
+git clone https://github.com/MetaMask/provider-engine
+# clone more repos...
```
Comment (danfinlay) [inline CONTRIBUTING.md]: Might also be worth noting that you should `npm install` in each of those projects.

**Review by danfinlay — COMMENTED** [review]
This is probably an improvement worth merging already, but I added a couple notes that could be improvements.

**Comment by danfinlay** [comment]
Would you say this fixes #3780?

**Comment by lazaridiscom** [comment]
> Would you say this fixes #3780?

no no, #3780 strives for a near-perfect solution (which will need most possibly an additional tool)

**Comment by lazaridiscom** [comment]
@danfinlay , this should be mergable.

**Review by danfinlay — APPROVED** [review]

**Comment by lazaridiscom** [comment]
@kumavis

**Comment by lazaridiscom** [comment]
@danfinlay , @kumavis , I've moved this over to the mm-vault project, so if there is any reason to not merge this now here, then you could close this.

**Pull request closed without merging**